docs/workflow_generators/residential_hpxml.rst

```diff
@@ -38,43 +41,62 @@ Configuration Example
 Arguments
 ~~~~~~~~~
 
-- ``build_existing_model``: Update the simulation control arguments to the `BuildExistingModel`_ measure. See
-  :ref:`build-existing-model-defaults` for current defaults.
+- ``build_existing_model``: Update the simulation control arguments to the `BuildExistingModel`_ measure.
+  See :ref:`build-existing-model-defaults` for current defaults.
+
```

Comment:
I don't see the default timestep listed.

Reply:
You can view the built documentation as an Actions artifact: https://github.com/NREL/buildstockbatch/actions/runs/2157754281.